connect/runtime/src/test/java/org/apache/kafka/connect/runtime/WorkerTest.java

```diff
@@ -110,6 +113,37 @@ public void setup() {
         PowerMock.mockStatic(Plugins.class);
     }
 
+    @Test
+    public void testLookupKafkaClusterId() {
+        final Node broker1 = new Node(0, "dummyHost-1", 1234);
+        final Node broker2 = new Node(1, "dummyHost-2", 1234);
+        List<Node> cluster = new ArrayList<Node>(2) {
```

Comment:
I have to admit, I'm not a big fan of the trick with the anonymous class on collections. 

`MockAdminClient` doesn't seem to mutate this argument, plus it should probably create a copy if it did anyways. 

Given that, `List<Node> cluster = Arrays.asList(broker1, broker1)` would be my suggestion, but I also like to leave plenty of room for different styles, so not a strong suggestion.

Reply:
I agree that `Arrays.asList(broker1, broker2)` is more readable and more concise.

Reply:
was just copy paste from another test, cleaned up.